src/main/native/glue/a/AaBox.cpp

```diff
@@ -112,6 +127,72 @@ JNIEXPORT void JNICALL Java_com_github_stephengold_joltjni_AaBox_encapsulate
     pBox->Encapsulate(location);
 }
 
+/*
+ * Class:     com_github_stephengold_joltjni_AaBox
+ * Method:    encapsulateBoundingBox
+ * Signature: (JJ)V
+ */
+JNIEXPORT void JNICALL Java_com_github_stephengold_joltjni_AaBox_encapsulateBoundingBox
+  (JNIEnv *, jclass, jlong boxVal, jlong rhs) {
+    AABox * const pBox = reinterpret_cast<AABox *> (boxVal);
+    AABox * const pRHS = reinterpret_cast<AABox *> (rhs);
```

Comment:
In this case, `pRHS` should be a `const AABox *` but `pBox` shouldn't. Please add `const` for clarity.